ethereum/nft/trades/trades.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE nft.trades (
+    block_time timestamptz NOT NULL,
+    nft_project_name text,
+    nft_token_id text,
+    platform text NOT NULL,
+    platform_version text,
+    category text,
+    evt_type text,
+    usd_amount numeric,
+    seller bytea,
+    buyer bytea,
+    original_amount numeric,
+    original_amount_raw numeric,
+    original_currency text,
+    original_currency_contract bytea,
+    currency_contract bytea,
+    nft_contract_address bytea NOT NULL,
+    exchange_contract_address bytea NOT NULL,
+    tx_hash bytea NOT NULL,
+    block_number integer,
+    tx_from bytea NOT NULL,
+    tx_to bytea,
+    trace_address integer[],
+    evt_index integer,
+    trade_id integer
+);
+
+CREATE UNIQUE INDEX CONCURRENTLY IF NOT EXISTS nft_trades_platform_tx_hash_evt_index_uniq_idx ON nft.trades (platform, tx_hash, evt_index);
```

Comment:
Is there a reason you don't have `project, tx_hash, trace_address, trade_id` uniquely indexed?

Reply:
No I do not.You make a great point, added in new commit.

Reply:
👍